docs/calico-with-docker/AWS.md

```diff
@@ -190,33 +190,38 @@ aws ec2 run-instances \
   --key-name mykey \
   --security-group-ids $SECURITY_GROUP_ID \
   --user-data file://<PATH_TO_CLOUD_CONFIG>/user-data-others
-#  --subnet $SUBNET_ID 
+#  --subnet $SUBNET_ID
 #  Include the subnet param above if using a non-default VPC
 ```
 
+Finally, disable `Source/Dest. Check` to allow containers to talk between hosts.  You can disable this with the CLI, or in the `Networking` part of the instances' right click menus.  Run the following for each server:
+```
+aws ec2 modify-instance-attribute --instance-id <INSTANCE_ID> --source-dest-check "{\"Value\": false}"
```

Comment:
It doesn't look like we explain where to get <INSTANCE_ID> from - we should probably mention where that value comes from.

Also, we should use `$INSTANCE_ID` so that this command can be copied / pasted.